docs/resources/workspace_service.md

```diff
@@ -0,0 +1,197 @@
+---
+subcategory: "Workspace"
+---
+
+# huaweicloud_workspace_service
+
+Use this resource to subscribe or unregistry the Workspace service in HuaweiCloud.
```

Comment:
use 'register or unregister' could be better

Reply:
done.